### DISCUSSION TITLE
Cross datasource query

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#9040] [Help]
Is there a way to make cross datasource queries ?

My main DB is a postgreSQL, but I have the users in the inner db, and also attachment in another bb table.
I'd like to be able to make queries joining for example attachments and postgreSQL data...

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

There's no direct way to do that. Your only option would be to have "soft" relationships between tables of different datasources, and use filters in the **Design** section.
